### ISSUE TITLE
Best way to add negative Tax

### ISSUE THREAD

**Issue by restos-99** [bytefury/crater#315]
I need to create a negative Tax to subtract certain % from  invoice subtotal. Which is the best way to do it?

thanks.

**Comment by mohitpanjwani** [comment]
@restos-99 Hey, I don't think it's currently possible to create a negative tax. 

I've personally never had to do this. How often do you need to do this?

**Comment by ggiak** [comment]
@restos-99  is it possible that you need a discount instead of negative tax?

I don't think there is a negative tax in any book keeping worldwide.

**Comment by sal1ner0** [comment]
@restos-99 you're right. Don't know if is your case, but i need "negative tax" for Spain -> The IRPF (“Impuesto sobre la Renta de Personas Físicas“) is an earnings tax for all “physical people” (i.e. not companies, associations or other entities). It is the equivalent of income tax in Spain.
